erica_app/tests/elster_xml/test_elster_xml_generator.py

```diff
@@ -710,9 +706,109 @@ def test_add_nutzdaten_header(self):
         self.assertIn("<NutzdatenTicket>nutzdatenTicket123</NutzdatenTicket>", xml_string)
         self.assertIn('<Empfaenger id="F">9198</Empfaenger>', xml_string)
 
+
+class TestGenerateFullEstXML(unittest.TestCase):
```

Comment:
I've split some of these tests into to separate classes because there was quite some stuff mixed together